entrypoint.sh

```diff
@@ -13,21 +13,24 @@ ssh-keyscan "$HOST" >> ~/.ssh/known_hosts
 
 cd "$GITHUB_WORKSPACE" || exit
 
-DEFAULT_BRANCH=${DEFAULT_BRANCH:-'master'}
+DEFAULT_BRANCH=${DEFAULT_BRANCH:-$GITHUB_PRIMARY_BRANCH}
 CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 if [[ $CURRENT_BRANCH == "$DEFAULT_BRANCH" && -n $DOMAIN ]]; then
     APP_NAME=$DOMAIN
 else
     APP_NAME=${CURRENT_BRANCH/\//-}
 fi
 
+echo "APP_NAME defined as $APP_NAME"
+
 echo "Checking if app exists"
-if ! ssh "dokku@$HOST" 'apps:exists $APP_NAME'; then
+if ! ssh "dokku@$HOST -p $PORT dokku apps:exists $APP_NAME"; then
     echo "The app does not exist yet, creating the app"
-    ssh "dokku@$HOST" 'apps:create $APP_NAME'
+    ssh "dokku@$HOST -p $PORT dokku apps:create $APP_NAME"
```

Comment:
```suggestion
    ssh "dokku@$HOST" -p $PORT dokku apps:create $APP_NAME
```

I test that when the closing quote is placed at the end of the file, the command doesn't work

Reply:
aah ok.. thanks! last minute moved the quotation mark.. doh! thanks!